README.fr.md

```
@@ -0,0 +1,613 @@
+[![npm downloads](https://img.shields.io/npm/dm/mulmoclaude.svg)](https://www.npmjs.com/package/mulmoclaude)
+[![License: MIT](https://img.shields.io/npm/l/mulmoclaude.svg)](LICENSE)
+[![CI](https://github.com/receptron/mulmoclaude/actions/workflows/pull_request.yaml/badge.svg)](https://github.com/receptron/mulmoclaude/actions/workflows/pull_request.yaml)
+[![GitHub stars](https://img.shields.io/github/stars/receptron/mulmoclaude.svg?style=social)](https://github.com/receptron/mulmoclaude/stargazers)
+
+[English](README.md) · [日本語](README.ja.md) · [简体中文](README.zh.md) · [한국어](README.ko.md) · [Español](README.es.md) · [Português (BR)](README.pt-BR.md) · **Français** · [Deutsch](README.de.md)
+
+Chat graphique avec Claude Code — avec mémoire à long terme, outils visuels et accès aux applications de messagerie.
+
+Discutez avec Claude Code et recevez non seulement du texte, mais aussi une **sortie visuelle interactive** : documents, feuilles de calcul, cartes mentales, graphiques, images, formulaires, scènes 3D et plus encore. Un wiki personnel intégré donne à Claude des **connaissances persistantes** qui s'enrichissent à chaque conversation.
+
+**Accédez depuis n'importe où** : connectez Telegram, Slack, LINE, Discord ou [10 autres applications de messagerie](#messaging-bridges) pour dialoguer avec votre agent IA depuis votre téléphone.
```

markdownlint-cli2 (0.22.0)

[warning] Link fragments should be valid

(MD051, link-fragments)